automation/devops_automation_infra/plugins/memsql.py

```diff
@@ -173,7 +174,8 @@ def fetch_count(self, query):
 
     def ping(self):
         try:
-            return self.fetch_all("show databases")
+            nodes_status = json.loads(self._host.Docker.run_cmd_in_service('memsql', 'gosu memsql memsql-admin list-nodes --json'))
+            return all([node['processState'] == 'Running' for node in  nodes_status['nodes']])
```

Comment:
great! in bash I do the same but uglier :)

Reply:
processState = Running 
Connectable = True
RecoveryState = Online

these are the correct states

Reply:
should i check all of those states ?

Reply:
ok i added the checks for all 3